src/notation/view/widgets/editstyle.cpp

```diff
@@ -1473,6 +1474,60 @@ QString EditStyle::pageCodeForElement(const EngravingItem* element)
     }
 }
 
+QString EditStyle::subPageCodeForElement(const EngravingItem* element)
+{
+    IF_ASSERT_FAILED(element) {
+        return QString();
+    }
+
+    switch (element->type()) {
+    case ElementType::TEXT: {
+
+        switch (toText(element)->textStyleType()) {
+        case TextStyleType::SUBTITLE:
+            return "subtitle";
+
+        case TextStyleType::COMPOSER:
+            return "composer";
+
+        case TextStyleType::LYRICIST:
+            return "poet";
+
+        case TextStyleType::TRANSLATOR:
+            return "translator";
+
+        case TextStyleType::FRAME:
+            return "frame";
+
+        case TextStyleType::HEADER:
+            return "header";
+
+        case TextStyleType::FOOTER:
+            return "footere";
+
+        default: return "title";    
+        }
+    }
+
+    case ElementType::INSTRUMENT_NAME: {
+        const mu::engraving::InstrumentName *instruName = toInstrumentName(element);
```

Comment:
The * should be at the end of `InstrumentName`, rather than at the start of `instruName`